cmd/crc/cmd/oc_env.go

```diff
@@ -23,16 +23,17 @@ var ocEnvCmd = &cobra.Command{
 			errors.ExitWithMessage(1, "Error running the oc-env command: %s", err.Error())
 		}
 
-		proxyConfig, err := machine.GetProxyConfig(constants.DefaultName)
+		clusterConfig, err := machine.GetClusterConfig(constants.DefaultName)
 		if err != nil {
 			errors.Exit(1)
 		}
 		output.Outln(shell.GetPathEnvString(userShell, constants.CrcBinDir))
-		if proxyConfig.IsEnabled() {
-			output.Outln(shell.GetEnvString(userShell, "HTTP_PROXY", proxyConfig.HttpProxy))
-			output.Outln(shell.GetEnvString(userShell, "HTTPS_PROXY", proxyConfig.HttpsProxy))
-			output.Outln(shell.GetEnvString(userShell, "NO_PROXY", proxyConfig.GetNoProxyString()))
+		if clusterConfig.ProxyConfig.IsEnabled() {
+			output.Outln(shell.GetEnvString(userShell, "HTTP_PROXY", clusterConfig.ProxyConfig.HttpProxy))
+			output.Outln(shell.GetEnvString(userShell, "HTTPS_PROXY", clusterConfig.ProxyConfig.HttpsProxy))
+			output.Outln(shell.GetEnvString(userShell, "NO_PROXY", clusterConfig.ProxyConfig.GetNoProxyString()))
 		}
+		output.Outf("echo \"You can now login to the OpenShift cluster with 'oc login -p developer -u developer %s'\"\n", clusterConfig.ClusterAPI)
```

Comment:
@cfergeau only for developer user or also for admin?

Reply:
Not sure we should advertise the admin user too strongly?

Reply:
@cfergeau most of the user who want to experiment with operators like making any config changes or installing a new one require the the admin even with operator hub so I think it should be good to have both user mentioned here like we do for `crc console credentials`.

Reply:
But if we push too much the admin credentials, won't crc users always use that, and just ignore the `developer` user?

Reply:
@cfergeau it's just user choice we should provide them both the options like we are providing for other user facing commands.